# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextProviderWGL.cpp
# Commit: 0067bb1fb8e4
# Full Hash: 0067bb1fb8e45078f4383c9e371dd1c7eafbffbe
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2020-07-29 09:49:32
# Regressor Bug: 1652972
# File Overlap Count: 1
# Description:
#   Bug 1652972 - Mark EGL context as lost on Shutdown() r=aosmond
#   
#   This is meant to save us in cases where the message loop in GPU process
#   receives commands related to resources that point to the old EGL context
#   that was just shut down. Since the symbols are erased, we'd end up with
# ==============================================================================

diff -r bd33c30ba4b5 -r 0067bb1fb8e4 gfx/gl/GLContextProviderWGL.cpp
--- a/gfx/gl/GLContextProviderWGL.cpp	Tue Jul 28 22:26:21 2020 +0000
+++ b/gfx/gl/GLContextProviderWGL.cpp	Wed Jul 29 02:33:54 2020 +0000
@@ -298,6 +298,8 @@
   }
 }
 
+bool GLContextWGL::IsAliveImpl() const { return true; }
+
 bool GLContextWGL::MakeCurrentImpl() const {
   const bool succeeded = sWGLLib.mSymbols.fMakeCurrent(mDC, mContext);
   NS_ASSERTION(succeeded, "Failed to make GL context current!");